site/packages/aa-ethers/provider-adapter/constructor.md

```diff
@@ -35,24 +35,24 @@ export const provider = new EthersProviderAdapter({
 });
 
 // another way to initialize
-const accountProvider = new AlchemyProvider({
+export const smartAccountClient = createAlchemySmartAccountClient({
   apiKey: "ALCHEMY_API_KEY", // replace with your Alchemy API Key
   chain: sepolia,
-  entryPointAddress: getDefaultEntryPointAddress(sepolia),
   opts: {
     txMaxRetries: 10,
     txRetryIntervalMs: 2_000,
     txRetryMultiplier: 1.5,
     minPriorityFeePerBid: 100_000_000n,
-  },
-  feeOpts: {
-    baseFeeBufferPercent: 50n,
-    maxPriorityFeeBufferPercent: 5n,
-    preVerificationGasBufferPercent: 5n,
+    feeOpts: {
+      baseFeeBufferPercent: 50n,
+      maxPriorityFeeBufferPercent: 5n,
+      preVerificationGasBufferPercent: 5n,
+    },
   },
 });
+
 export const anotherProvider = new EthersProviderAdapter({
-  accountProvider,
+  smartAccountClient,
```

Comment:
this should stay the way it is since that's the name of the property in the `EthersProviderAdapter`, or we should update the property on the `EthersProviderAdapter` to match this name